packages/cli/src/lib/cache/SuccessCache.ts

```diff
@@ -24,6 +25,15 @@ const CACHE_MAX_AGE_MS = 7 * 24 * 3600_000;
 export class SuccessCache {
   readonly #path: string;
 
+  /**
+   * Trim any occurrences of the workspace root path from the input string. This
+   * is useful to ensure stable hashes that don't vary based on the workspace
+   * location.
+   */
+  static trimPaths(input: string) {
+    return input.replaceAll(paths.targetRoot, '');
```

Comment:
hmf, shouldn't it do something smart with specifically the start of the string? :)

Reply:
you mean trim from the start only? This is a lot more coarse than that, all existing usages is trimming stuff that's JSON serialized first